build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1961,9 +1961,30 @@ message Compressor {
 
     // Brotli compression.
     BROTLI = 3;
+
+    // Zstandard compression with dictionary.
+    //
+    // When this is used, the server MUST advertise the dictionaries by
+    // including
+    // [ZstdDictionaryRegistry][build.bazel.remote.execution.v2.ZstdDictionaryRegistry]
+    // digest in CacheCapabilities.
+    ZSTD_DICT = 4;
   }
 }
 
+message ZstdDictionaryRegistry {
+  // Each file respresent a single Zstandard dictionary
+  // with name being the `dictId` that was added to the header
+  // of the compressed file.
+  //
+  // A special file with name `default` is used to represent the
+  // default dictionary that is used when no `dictId` is specified.
+  // Clients SHOULD prefer to use the default dictionary when possible.
+  //
+  // The `default` dictionary MUST be present.
+  repeated FileNode dictionaries = 1;
```

Comment:
FileNode has a bunch of unrelated and unspecified fields.

I wonder if it would be simpler to have a repeated Digest field, and a separate "Default" field which is an index into the array/slice/whatever of the digests? Or maybe just a separate "Default" digest?